bin/bb8/docker_rsync.py

```diff
@@ -86,6 +86,21 @@ def _create_target_dirs(self, host, target_path):
         self._make_remote_dir(host, join(target_path, "data"))
         self._make_remote_dir(host, join(target_path, "meta"))
 
+    def _validate_instance(self, host, name, target_path, instance_guid):
+        remote_id = self._run(command=["ssh", host, "less", target_path],
```

Comment:
I presume that this is coming out with @MartinEden's work but I believe this should be `cat` not `less`